### PR TITLE
Update golang version to 1.9.x

### LINKED ISSUE #408: Update golang version to 1.9.x
Kubernetes 1.9 updated golang version to 1.9.1.
https://github.com/kubernetes/kubernetes/pull/55301

### PR DESCRIPTION
**Pull request by miaoyq**
fixes #408

Signed-off-by: Yanqiang Miao <miao.yanqiang@zte.com.cn>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ jobs:
         - make .gitvalidation
         - make verify
         - make binaries
-      go: 1.8.x
+      go: 1.9.x
```
Comment (mikebrow) [inline .travis.yml]: we could build both for a while..

Comment (miaoyq) [reply .travis.yml]: Done.

File: .travis.yml [hunk 2]
```diff
@@ -56,6 +56,12 @@ jobs:
         - make .gitvalidation
         - make verify
         - make binaries
+      go: 1.9.x
```
Comment (Random-Liu) [inline .travis.yml]: Let's just remove 1.8.x
Both Kubernetes and containerd moved to 1.9 already.

**Comment by Random-Liu** [comment]
LGTM with one nit.

File: .travis.yml (hunk 2 above)
Comment (miaoyq) [reply .travis.yml]: Done.

File: .travis.yml (hunk 1 above)
Comment (Random-Liu) [reply .travis.yml]: @mikebrow Sorry, just saw your comment.

Actually both containerd and Kubernetes have moved to 1.9 already. I think we should build with 1.9. :)

Comment (mikebrow) [reply .travis.yml]: Ok fair enough, time to move up then... waves good bye to 1.8

**Pull request merged**